pages/linux/grubby.md

```diff
@@ -0,0 +1,16 @@
+# grubby
+
+> Command line tool for configuring grub and zipl.
```

Comment:
```suggestion
> Tool for configuring `grub` and `zipl`.
```
We generally don't use wording like "Command line tool" at the start and instead prefer just tool, etc. And we enclose commands inside backticks.